src/clj/athens/self_hosted/components/config.clj

```diff
@@ -13,8 +13,11 @@
 
   (start
     [component]
-    (log/info "Starting configuration component")
-    (assoc component :config (cfg/reload-env)))
+    (let [config        (cfg/reload-env)
+          merged-config (cfg/merge-maps config (:config-edn config))]
+      (log/info "Starting configuration component")
+      (log/info merged-config)
```

Comment:
Nit: this isn't needed anymore, the final config is already printed elsewhere.

Reply:
There is no other `log/info` for config other than this.

Reply:
Thought I had seen repeat prints while we were debugging, lgtm then